onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorMatMulIntegerToFloat.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "precomp.h"
+
+namespace Dml
+{
+
+class DmlOperatorMatMulIntegerToFloat : public DmlOperator
+{
+    enum InputTensors : uint32_t { 
+        IN_A,
+        IN_B,
+        IN_A_SCALE,
+        IN_B_SCALE,
+        IN_A_ZERO_POINT, 
+        IN_B_ZERO_POINT,
+        IN_BIAS,
+        IN_COUNT
+    };
+    
+    enum DmlInputIndex : uint32_t
+        {
+            dmlA,
+            dmlAScale,
+            dmlAZeroPoint,
+            dmlB,
+            dmlBScale,
+            dmlBZeroPoint,
+            dmlBias,
+            dmlInputCount,
+        };
```

Comment:
```suggestion
    enum InputTensors : uint32_t
    { 
        IN_A,
        IN_B,
        IN_A_SCALE,
        IN_B_SCALE,
        IN_A_ZERO_POINT, 
        IN_B_ZERO_POINT,
        IN_BIAS,
        IN_COUNT
    };

    enum DmlInputIndex : uint32_t
    {
        dmlA,
        dmlAScale,
        dmlAZeroPoint,
        dmlB,
        dmlBScale,
        dmlBZeroPoint,
        dmlBias,
        dmlInputCount,
    };
```

Please keep consistent style. Even immediately adjacent code blocks are using different styles 🙃.